src/api.v2/helpers/pipeline/gem2s.js

```diff
@@ -0,0 +1,173 @@
+const _ = require('lodash');
+const AWSXRay = require('aws-xray-sdk');
+
+const constants = require('./constants');
+const getPipelineStatus = require('./getPipelineStatus');
+const { createGem2SPipeline } = require('./pipelineConstruct');
+
+const Sample = require('../../model/Sample');
+const Experiment = require('../../model/Experiment');
+const ExperimentExecution = require('../../model/ExperimentExecution');
+
+const sendNotification = require('./hooks/sendNotification');
+const HookRunner = require('./hooks/HookRunner');
+
+const validateRequest = require('../../../utils/schema-validator');
+const getLogger = require('../../../utils/getLogger');
+
+const logger = getLogger('[Gem2sService] - ');
+
+const hookRunner = new HookRunner();
+
+const saveProcessingConfigFromGem2s = ({ experimentId, item }) => {
+  logger.log('Saving processing config for gem2s');
+  new Experiment().updateById(experimentId, { processing_config: item });
+  logger.log('Finished saving processing config for gem2s');
+};
+
+const continueToQC = () => {
+  // const continueToQC = (payload) => {
```

Comment:
what is this?

Reply:
The part that will be implemented when we implement qc, I could remove the comments anyways